tests/TryCatchTest.php

```diff
@@ -369,13 +368,6 @@ class_alias(
                         }
                     }'
             ],
-            'suppressUndefinedVarInFinally' => [
```

Comment:
does this cause an issue?

Reply:
I think I removed that thinking it was testing a false negative that I fixed, but I'm not sure why I thought that. It seems to run fine now, I'll re-add it.